prompting/datasets/random_website.py

```diff
@@ -0,0 +1,63 @@
+import random
+from typing import Optional
+from duckduckgo_search import DDGS
+import trafilatura
+from prompting.datasets.base import BaseDataset, Context, DatasetEntry
+from loguru import logger
+
+
+def load_words(file_path: str) -> list[str]:
+    with open(file_path, "r") as file:
+        return file.read().splitlines()
+
+
+MAX_CHARS = 5000
+WORDS = load_words("prompting/datasets/english_words.txt")
```

Comment:
It will load the file everytime the `random_website` is imported, why not to put inside DDGDataset